simtools/data_model/format_checkers.py

```diff
@@ -0,0 +1,25 @@
+"""Custom format checkers for jsonschema validation."""
+
+import astropy.units as u
+import jsonschema
+
+from simtools.utils import names
+
+format_checker = jsonschema.FormatChecker()
+
+
+@format_checker.checks("astropy_unit", raises=ValueError)
+def check_astropy_unit(unit_string):
+    """Validate astropy units (including dimensionless) for jsonschema."""
+    try:
+        u.Unit(unit_string)
+        return True
+    except ValueError:
+        return unit_string == "dimensionless"
+
+
+@format_checker.checks("array_element", raises=ValueError)
```

Comment:
If I understood correctly, it would actually raise `ValidationError` and not `ValueError`, no?

Reply:
Yes, it raises ValueError. Motivation is to keep to standard python Exceptions and not to introduce the complex json schema exceptions

Reply:
Yeah I saw later and didn't delete this comment.